packages/artillery/lib/platform/aws-ecs/legacy/run-cluster.js

```diff
@@ -347,6 +347,38 @@ async function tryRunCluster(scriptPath, options, artilleryReporter) {
       process.exit(1);
     }
     options.launchConfig = launchConfig;
+  } else {
+    options.launchConfig = {};
+  }
+
+  if (options.cpu) {
+    const n = Number(options.cpu);
+    if (isNaN(n)) {
+      artillery.log('The value of --cpu must be a number');
+      process.exit(1);
+    }
+
+    // Allow specifying 16 vCPU as either "16" or "16384". The actual value is
+    // validated later.
+    if (n <= 16) {
```

Comment:
nit: can we put the number 16 as a const somewhere, rather than being a magic number? There's a comment, but still took me a sec to think "oh yeah, that's the max vCPU value currently available"

Reply:
yes good shout

Reply:
```suggestion
    const MAX_VCPUS = 16;
    if (n <= MAX_VCPUS) {
```